packages/i18n/locales/en.json

```
@@ -168,7 +168,8 @@
       "overview": "Overview",
       "minorVersions": "Minor versions",
       "releaseAnnouncement": "Release Announcement",
-      "unsupportedVersionWarning": "This version is out of maintenance. Please use a currently supported version."
+      "unsupportedVersionWarning": "This version is out of maintenance. Please use a currently supported version. <link>Understand EOL support.</link>",
+      "ltsVersionFeaturesNotice": "Want new features sooner? Get <link>Latest Node.js</link> instead and try the latest improvements on Node!"
```

```suggestion
      "ltsVersionFeaturesNotice": "Want new features sooner? Get the <link>latest Node.js version</link> instead and try the latest improvements on Node!"
```
I think this is more grammatically correct?